src/Microsoft.DotNet.Wpf/tests/UnitTests/System.Xaml.Tests/System/Xaml/Permissions/XamlAccessLevelTests.cs

```diff
@@ -0,0 +1,152 @@
+#if NETFRAMEWORK
```

Comment:
@hughbe Do we need to run these tests on .NET Core, since they are specifically designed for .NET Framework? Additionally, can we omit this flag altogether to run these tests on .NET Core?

Reply:
@hughbe any comments?

Reply:
@hughbe can we remove this test as of now to get the remaining one's added? We can get it unblocked later if needed.

Reply:
Sorry! Will remove tonight